config/environments/development.rb

```diff
@@ -14,9 +14,10 @@
   config.action_controller.perform_caching = false
   config.serve_static_files                = false
 
-  # Don't care if the mailer can't send.
+  # Do care if the mailer can't send.
```

Comment:
Would you mind reverting this back to don't, since `raise_delivery_errors` is still set to false? Don't want us to be rooting around in here six months from now and be like 'wtf'.